src/iceberg/v1_metadata.cc

```diff
@@ -47,15 +50,14 @@ Status ManifestEntryAdapterV1::Init() {
       DataFile::kSplitOffsets.field_id(),
       DataFile::kSortOrderId.field_id(),
   };
-  // TODO(xiao.dong) schema to json
-  metadata_["schema"] = "{}";
-  // TODO(xiao.dong) partition spec to json
-  metadata_["partition-spec"] = "{}";
+  ICEBERG_RETURN_UNEXPECTED(InitSchema(kManifestEntryFieldIds));
+  metadata_["schema"] = ToJson(*manifest_schema_).dump();
```

Comment:
```suggestion
  metadata_["schema"] = ToJsonString(ToJson(*manifest_schema_));
```

Reply:
Or we can add the following to `json_internal.h`

```cpp
template <typename T>
ICEBERG_EXPORT Result<std::string> ToJsonString(const T& value)
  requires requires { ToJson(value); }
{
  return ToJsonString(ToJson(value));
}
```

Reply:
I really think we should make toJson like this. This way we can ignore the nlohmann:: outside.

Reply:
I just add a new function: 
ICEBERG_EXPORT Result<std::string> ToJsonString(const Schema& schema);